wsd/ClientRequestDispatcher.cpp

```diff
@@ -1113,8 +1114,13 @@ bool ClientRequestDispatcher::handleClipboardRequest(const Poco::Net::HTTPReques
             tag = it.second;
         else if (it.first == "MimeType")
             mime = it.second;
+        else if (it.first == "charset")
+            charset = it.second;
     }
 
+    if (!charset.empty())
+        mime += ";charset=" + charset;
```

Comment:
I assume that the charset is then parsed by core, since you only produce this charset info, you don't add code to handle it somewhere. If this is the case, we could have a comment here stating this.

Reply:
It's handled below in line ~1167:

` else if (mime == "text/html,text/plain;charset=utf-8")`